apps_accessors.go

```diff
@@ -2886,38 +2886,6 @@ func (i *ImageSourceSpecDeployOnPush) GetEnabled() bool {
 	return i.Enabled
 }
 
-// GetAppID returns the AppID field.
```

Comment:
Yes, I dropped `UpgradeBuildpackRequest` in favor of `UpgradeBuildpackOptions`. I figured the change was ok since the UpgradeBuildpack operation was never implemented before.